app/src/js/components/home.js

```diff
@@ -147,6 +147,34 @@ class Home extends React.Component {
     }
   }
 
+  pageSection (identifier, title, children) {
+    return (
+      <section className={`page__section ${identifier}`}>
+        <div className='row'>
+          <div className='heading__wrapper'>
+            <h2 className='heading--medium heading--shared-content--right'>
+              {title}
+            </h2>
+          </div>
+          {children}
+        </div>
+      </section>
+    );
+  }
+
+  sectionHeader (identifier, title, link) {
+    return (
+      <section className={`page__section ${identifier}`}>
+        <div className='row'>
+          <div className='heading__wrapper--border'>
+            <h2 className='heading--large heading--shared-content--right'>{title}</h2>
+            {link}
+          </div>
+        </div>
+      </section>
```

Comment:
This is basically the same as a section, but with a --border and large heading. ripe for fixing.